cetmix_tower_server/models/cx_tower_reference_mixin.py

```diff
@@ -129,7 +128,8 @@
         """
         if not self._context.get("reference_mixin_override"):
             for vals in vals_list:
-                vals["name"] = vals["name"].strip()
+                if vals.get("name"):
+                    vals["name"] = vals["name"].strip()
```

Comment:
Thank you for pointing this out! After consideration, I realized that calling strip() on an empty string has no effect—it will remain empty. Therefore, we don't need to modify the condition to handle empty strings explicitly. The current condition if vals.get("name"): is sufficient, as it ensures strip() is only called when name has a meaningful, non-empty value.

This approach also skips any unnecessary processing on empty or None values, improving efficiency.